### ISSUE TITLE
Building a installer for windows

### ISSUE THREAD

**Issue by CasperHK** [visualfc/liteide#642]
If there is a windows installer, things will become more convenient.

**Comment by visualfc** [comment]
Decompression can be used

**Comment by CasperHK** [comment]
Yes, I know it but if there is a installer and a default installation path, it is good for management and it gives a better feeling.

**Comment by shyrobbiani** [comment]
@CasperHK: sorry, to jump in, but after I've been using and installing LiteIDE on both, Windows and Ubuntu, I really don't understand why you need an installer. I assume you read the [Install instructions in the Wiki ](https://github.com/visualfc/liteide/blob/master/liteidex/deploy/welcome/en/install.md). I like the way it is much more than any installer that does some magic in the background I don't understand. It fits perfectly into the concepts of Go.
